examples/previous-next-navigation/static.config.js

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios'
+
+export default {
+  getSiteData: () => ({
+    title: 'React Static',
+  }),
+  getRoutes: async () => {
+    const { data: posts } = await axios.get('https://jsonplaceholder.typicode.com/posts')
+    return [
+      {
+        path: '/',
+        component: 'src/containers/Home',
+      },
+      {
+        path: '/about',
+        component: 'src/containers/About',
+      },
+      {
+        path: '/blog',
+        component: 'src/containers/Blog',
+        getData: () => ({
+          posts,
+        }),
```

Comment:
Similarly, you could move `Blog.js` to `pages/blog.js` and then remove the `component` line. You would however need to keep the `path` and `getData`, to properly tie the data to that route.